config/development.json

```diff
@@ -0,0 +1,15 @@
+{
+  "baseURL": "http://api.127.0.0.1.xip.io:3030",
+  "frontURL": "http://webapp.127.0.0.1.xip.io:3030",
+  "mongodb": "mongodb://mongo:27017/hc_api",
+  "host": "0.0.0.0",
+  "authentication": {
+    "jwt": {
+      "audience": "http://webapp.127.0.0.1.xip.io:3030"
+    }
+  },
+  "seeder": {
```

Comment:
And maybe remove `seeder` part completely. It should be up to the user if he or she wants to seed the database in docker.

Reply:
changed what you suggested